### PR TITLE
Improve success message for sign in/out

### PR DESCRIPTION
**Pull request by codemist**
Implemented new design for the sign in/out success message. Figma reference: 

image

### CODE REVIEW

File: privaterelay/templates/home.html [hunk 1]
```diff
@@ -58,4 +60,5 @@ <h3 class="step-headline ff-Met">{% ftlmsg 'how-it-works-step-3-headline' %}</h3
 		</div>
 	</div>
 
+
```
Comment (groovecoder) [inline privaterelay/templates/home.html:63]:
```suggestion
```

File: privaterelay/templates/includes/header.html [hunk 2]
```diff
@@ -71,6 +71,7 @@
     </nav>
     {% include "includes/login.html" %}
   </div>
+
```
Comment (groovecoder) [inline privaterelay/templates/includes/header.html:74]:
```suggestion
```

File: privaterelay/templates/profile.html [hunk 3]
```diff
@@ -10,9 +10,11 @@
 
 {% get_social_accounts request.user as accounts %}
 
-{% include "includes/messages.html" with messages=messages %}
+<!--{% include "includes/messages.html" with messages=messages %}-->
```
Comment (groovecoder) [inline privaterelay/templates/profile.html:13]:
```suggestion
```

**Review by groovecoder — CHANGES_REQUESTED** [review]

**Comment by groovecoder** [comment]
Looks good but has some conflicts to resolve? Are you comfortable doing that yourself?

**Comment by codemist** [comment]
Looks fine here: 
image

I can get on a meeting with Maxx about resolving conflicts when he's back since I've not done that before.

**Comment by maxxcrawford** [comment]
Test

**Pull request closed without merging**